react/components/EXPERIMENTAL_Charts/BarChart/sampleData.js

```diff
@@ -0,0 +1,46 @@
+const data = [
+  {
+    name: 'Page A',
```

Comment:
Try to make the sample data more "ecommerce oriented". Maybe:
```
{
  name: "Home page"
},
{
  name: "Product page"
},
{
  name: "Category page"
}
```

Reply:
I applied your suggestion, but don't merge this, i need to add some changes

Reply:
Done!